cdn-server/cdn/src/index.ts

```diff
@@ -1,8 +1,9 @@
 import { JWTVerifyResult, jwtVerify } from 'jose';
 import { Context, Env, Hono, Next, Schema } from 'hono';
+import { streamToJSON } from './utils';
 
 export interface BlobStorage {
-  getObject(context: Context, key: string): Promise<ReadableStream>;
+  getObject(context: Context, key: string, cacheControl?: string): Promise<ReadableStream>;
```

Comment:
Let's work with a object here that is easier to extend.

Reply:
Can we also make it typesafe?

Reply:
you mean like cacheControl?: "no-cache" | ...  ?

we cant do that because the max-age can have any value

Reply:
Ok, let's go with any value.